PocketExample/Pocket/PocketProtocol.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  PocketProtocol.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 4/24/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+
+protocol Pocket {
+    func createWallet(passphrase: String) -> Wallet
+    func importWallet(walletKey: String, address: String) -> Wallet
+    func createTransaction(nonce: String, recipient: String, value: String, data: NSDictionary) -> Transaction
```

Comment:
Do not use an `NSDictionary`.

Reply:
To make it more modular I replaced it with Dictionary<AnyHashable, Any> as you mentioned earlier.